File: Python/Find-The-Sum/find_the_sum.py
import sys

if len(sys.argv) > 1:
	number = int(sys.argv[1])
	unique_pairs = []
	for i in range(1, 9):
		for j in range(i+1, 10):
			if i + j == number:
				unique_pairs.append((i, j))
	print(f'Input: {number}\nOutput: {len(unique_pairs)} --> {unique_pairs}')
else:
	print('Missing Argument')

File: C/Find-The-Sum/find-the-sum.c
#include <stdio.h>


int main(void)
{
	int len = 20, uniquePairs[len], counter = 0, number;

	printf("Input: ");
	scanf("%i", &number);

	for (int i = 0; i < len; i++)
	{
		uniquePairs[i] = 0;
	}

	for (int j = 1; j < 9; j++)
	{
		for (int c = j+1; c < 10; c++)
		{
			if ((j + c) == number)
			{
				uniquePairs[counter++] = j;
				uniquePairs[counter++] = c;
			}
		}	
	}

	printf("Output: %i --> ", (counter / 2));
	for (int m = 0; m < counter; m += 2)
	{
		printf("(%i, %i), ", uniquePairs[m], uniquePairs[m+1]);
	}
	printf("\n");

	return 0;
}